cognite_toolkit/_cdf_tk/client/data_classes/raw.py

```diff
@@ -56,3 +67,360 @@ class RawTableList(WriteableCogniteResourceList[RawTable, RawTable]):
 
     def as_write(self) -> RawTableList:
         return self
+
+
+@dataclass
+class RawProfileColumn(CogniteObject):
+    _type: ClassVar[str]
+    count: int
+    null_count: int
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> RawProfileColumn:
+        for column_type, column_class in _RAW_PROFILE_COLUMN_BY_TYPE.items():
+            if column_type in resource:
+                return column_class._load_column(resource)
+        return UnknownTypeProfileColumn._load_column(resource)
+
+    @classmethod
+    @abstractmethod
+    def _load_column(cls, resource: dict[str, object]) -> RawProfileColumn:
+        """Load a specific type of RawProfileColumn based on the resource data."""
+        raise NotImplementedError("Subclasses must implement this method.")
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        return {
+            "count": self.count,
+            "nullCount" if camel_case else "null_count": self.null_count,
+        }
+
+
+@dataclass
+class StringProfile(CogniteObject):
+    length_range: tuple[int, int]
+    distinct_count: int
+    length_histogram: tuple[tuple[int, int], ...]  # List of (length, count) tuples
+    value_counts: dict[str, int]
+    count: int
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> StringProfile:
+        return cls(
+            length_range=tuple(resource["lengthRange"]),
+            distinct_count=resource["distinctCount"],
+            length_histogram=tuple((length, count) for length, count in zip(*resource["lengthHistogram"])),
+            value_counts={value: count for value, count in zip(*resource["valueCounts"])},
+            count=resource["count"],
+        )
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        length_histogram: list[list[int]] = [[], []]
+        for length, count in self.length_histogram:
+            length_histogram[0].append(length)
+            length_histogram[1].append(count)
+        value_counts: list[list[str | int]] = [[], []]
+        for value, count in self.value_counts.items():
+            value_counts[0].append(value)
+            value_counts[1].append(count)
+
+        return {
+            "lengthRange" if camel_case else "length_range": list(self.length_range),
+            "distinctCount" if camel_case else "distinct_count": self.distinct_count,
+            "lengthHistogram" if camel_case else "length_histogram": length_histogram,
+            "valueCounts" if camel_case else "value_counts": value_counts,
+            "count": self.count,
+        }
+
+
+@dataclass
+class NumberProfile(CogniteObject):
+    value_range: tuple[float, float]
+    distinct_count: int
+    value_counts: dict[float, int]
```

Comment:
This is `Map[String, Int]` in the profiler backend, it looks like, are you sure `float` is valid?

Reply:
You are correct, it returns a string and not float.